ai_edge_torch/generative/layers/unet/model_config.py

```diff
@@ -173,6 +181,7 @@ class MidBlock2DConfig:
   transformer_block_config: Optional[TransformerBlock2DConfig] = None
   # Optional dimension of context tensor if context tensor is given as input.
   context_dim: Optional[int] = None
+  attention_hidden_channels: Optional[int] = 736
```

Comment:
`attention_hidden_channels: Optional[int] = None`

Reply:
Removed, this field was not used